src/pybind/mgr/influx/module.py

```diff
@@ -0,0 +1,211 @@
+import json
+import sys
+import time
+from ConfigParser import SafeConfigParser
+from collections import defaultdict
+from influxdb import InfluxDBClient
+from mgr_module import MgrModule
+from datetime import datetime
+from threading import Event 
+
+class Module(MgrModule):
+ 
+    run = True 
+    COMMANDS = [
+        {
+            "cmd": "influx debug",
```

Comment:
Could you call this "influx self-test" to match the similar command that we have in the zabbix module?

At some point we might want to generalize the self-test into some automated hook, so it would be neat to make them consistent as possible.